### PR TITLE
Astro 10/18 release notes

### PR DESCRIPTION
**Pull request by jwitz**
Based on https://app.zenhub.com/workspaces/developer-productivity-and-readiness-6242176c89ed9000197ec734/reports/release?release=Z2lkOi8vcmFwdG9yL1JlbGVhc2UvODMyMjc

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -15,6 +15,13 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.6.0 ([Release notes](cli/release-notes.md))
 
+## October 18, 2022 
+
+### Additional improvements 
+
+- The **Access** menu is now a tab in the **Workspace Settings** menu.
```
Comment (kwcanuck) [inline astro/release-notes.md:22]:
```suggestion
- **Access** has been moved from the left Cloud UI menu to a tab on the **Workspace Settings** page.
```
Has the new **Details** tab been mentioned previously?

File: astro/release-notes.md [hunk 2]
```diff
@@ -15,6 +15,13 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
...
+- The **Workspace Settings** is now visible to all Workspace members.
```
Comment (kwcanuck) [inline astro/release-notes.md:23]:
```suggestion
- **Workspace Settings** in the left menu of the Cloud UI is now available to all Workspace members.
```

**Review by kwcanuck — APPROVED** [review]

**Review by amccabe-astronomer — APPROVED** [review]
👍

File: astro/release-notes.md (hunk 1 above)
Comment (jwitz) [reply astro/release-notes.md:22]: Not directly, but we've discussed it indirectly as part of a previous release note in "A simpler Deployment page"

File: astro/release-notes.md [hunk 3]
```diff
@@ -15,6 +15,20 @@ If you have any questions or a bug to report, reach out to [Astronomer support](
 
 **Latest CLI Version**: 1.6.0 ([Release notes](cli/release-notes.md))
 
+## October 18, 2022 
+
+### Additional improvements 
+
+- In the Cloud UI, **Access** has been moved from the left menu to a tab on the **Workspace Settings** page.- In the Cloud UI, **Workspace Settings** in the left menu is now available to all Workspace members.
```
Comment (kwcanuck) [inline astro/release-notes.md:22]:
```suggestion
- In the Cloud UI, **Access** has been moved from the left menu to a tab on the **Workspace Settings** page.
- In the Cloud UI, **Workspace Settings** in the left menu is now available to all Workspace members.
```

**Pull request merged**